shared/middleware/helpers/storage_apps.rb

```diff
@@ -57,14 +62,14 @@ def update(channel_id, value, ip_address)
     update_count = @table.where(id:id).exclude(state:'deleted').update(row)
     raise NotFound, "channel `#{channel_id}` not found" if update_count == 0
 
-    JSON.parse(row[:value]).merge(id: channel_id, isOwner: owner == @storage_id)
+    JSON.parse(row[:value]).merge(id: channel_id, isOwner: owner == @storage_id, createdAt: row[:created_at], updatedAt: row[:updated_at])
```

Comment:
I took a stab at implementing `createdAt` in https://github.com/code-dot-org/code-dot-org/commit/73f41c70adef0eed97b6c3046548ec6726533397 but it requires an extra database query.  Since most calls to `update` won't do anything with the `createdAt` field I don't plan to merge this commit.

Followed up with @davidsbailey offline and it sounds we could get rid of the response from `update` in the future and just return the timestamp, or `200 No Content`.